metabase/metabase
# Adding flex grow to card wrapper

Fixes #28726 
Reproduces #28726 

Issue was caused by the preview not being given a proper height. We recently added a wrapper around the `CardVisualization` component, but the parent component for previews assumes that it's direct child will have `flex-grow` applied to use up the space.

How to test:
- Start a new question
- Sample dataset -> Orders
- Click the preview button, you should see a preview as a table.

![image](https://user-images.githubusercontent.com/1328979/221831007-75a2f958-5071-485c-9ec9-97c790e1e478.png)
